[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #60249 [ONNX] Update documentation (#58712)
* #60248 [ONNX] shape type inference fixes for control flow (#59319)
* #60247 [ONNX] Extend chunk for dynamic chunk values (#59644)
* #60246 [ONNX] Add linspace symbolic (#58854)
* #60245 [ONNX] Fix sum export with attribute keepdims (#59316)
* #60244 [ONNX] Fix shape inference for large model (#59320)
* **#60243 [ONNX] Handle onnx::Size in ComputeConstant folding (#59122)**

Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

Differential Revision: [D29494915](https://our.internmc.facebook.com/intern/diff/D29494915)

jiafatom committed:
[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)

Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

[ghstack-poisoned]

SplitInfinity commented:
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29494915).

BowenBao committed:
Update on "[ONNX] Handle onnx::Size in ComputeConstant folding (#59122)"


Handle onnx::Size in ComputeConstant folding

Co-authored-by: jiafatom <jiafa@microsoft.com>

Differential Revision: [D29494915](https://our.internmc.facebook.com/intern/diff/D29494915)

[ghstack-poisoned]

SplitInfinity commented:
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29494915).
